nntrainer/src/model_loader.cpp

```diff
@@ -240,8 +242,10 @@ int ModelLoader::loadFromIni(std::string ini_file, NeuralNetwork &model) {
   status = loadModelConfigIni(ini, model);
   NN_INI_RETURN_STATUS();
 
-  status = loadDatasetConfigIni(ini, model);
-  NN_INI_RETURN_STATUS();
+  if (model.isTrain()) {
```

Comment:
ditto

Reply:
this databuffer is needed when train mode.